app/src/main/java/gov/va/vro/security/ApiAuthKeyManager.java

```diff
@@ -27,17 +31,48 @@ public void setApiAuthKeys(ApiAuthKeys apiAuthKeys) {
     this.apiAuthKeys = apiAuthKeys;
   }
 
+  public JwtValidator getJwtValidator() {
+    return jwtValidator;
+  }
+
+  @Autowired
+  public void setJwtValidator(JwtValidator jwtValidator) {
```

Comment:
Let's use constructor injection: Make the fields final and use @RequiredArgumentsConstructor

Reply:
used setter injection as apikeys or jwt are optional based on the version of the API